tuxemon/monster.py

```diff
@@ -549,6 +552,27 @@ def set_moves(self, level: int) -> None:
                 tech.load(ele)
                 self.learn(tech)
 
+    def update_moves(self, levels: int) -> Optional[Technique]:
+        """
+        Set monster moves according to the levels increased.
+        Excludes the moves already learned.
+
+        Parameters:
+            level: Nr of levels.
```

Comment:
Inconsistency between docstring parameter and actual implementation parameter name (level/levels).

Reply:
The part about the returned value is also missing.

Reply:
Fixed